src/models/data.cpp

```diff
@@ -133,6 +133,158 @@ ItemCost buildCallerCalleeResult(const BottomUp& data, const Costs& bottomUpCost
+            }
+            else if (type == QLatin1String("wchar_t")) {
+                result += QLatin1String("wstring");
+            }
+            else {
+                result += symbol.left(end);
+                result += type;
+                result += QLatin1Char('>');
+            }
+            end = findSameDepth(symbol, 0, QLatin1Char('>'), true);
+            symbol = symbol.mid(end);
+
+            if (startsWith(symbol, QLatin1String("::basic_string("), &end)
+                || startsWith(symbol, QLatin1String("::~basic_string("), &end)) {
+                result += QLatin1String("::");
+                if (symbol[2] == QLatin1Char('~')) {
+                    result += QLatin1Char('~');
+                }
+                if (type == QLatin1String("wchar_t")) {
+                    result += QLatin1Char('w');
+                }
+                else if (type != QLatin1String("char")) {
+                    result += QLatin1String("basic_");
+                }
+                result += QLatin1String("string(");
+                symbol = symbol.mid(end);
+            }
+        }
+    }
+    else if (startsWith(symbol, QLatin1String("vector<"), &end)
+             || startsWith(symbol, QLatin1String("set<"), &end)
+             || startsWith(symbol, QLatin1String("deque<"), &end)
+             || startsWith(symbol, QLatin1String("list<"), &end)
+             || startsWith(symbol, QLatin1String("forward_list<"), &end)
+             || startsWith(symbol, QLatin1String("multiset<"), &end)
+             || startsWith(symbol, QLatin1String("unordered_set<"), &end)
+             || startsWith(symbol, QLatin1String("unordered_multiset<"), &end)) {
+        const int comma = findSameDepth(symbol, end, QLatin1Char(','));
+        if (comma != -1) {
+            result += symbol.left(end);
+            result += prettifySymbol(symbol.mid(end, comma - end));
+            result += QLatin1Char('>');
+
+            end = findSameDepth(symbol, 0, QLatin1Char('>'), true);
+            symbol = symbol.mid(end);
+        }
+    }
+    else if (startsWith(symbol, QLatin1String("map<"), &end)
+             || startsWith(symbol, QLatin1String("multimap<"), &end)
+             || startsWith(symbol, QLatin1String("unordered_map<"), &end)
+             || startsWith(symbol, QLatin1String("unordered_multimap<"), &end)) {
+        const int comma1 = findSameDepth(symbol, end, QLatin1Char(','));
+        const int comma2 = findSameDepth(symbol, comma1 + 1, QLatin1Char(','));
+        if (comma1 != -1 && comma2 != -1) {
+            result += symbol.left(end);
+            result += prettifySymbol(symbol.mid(end, comma1 - end));
+            result += prettifySymbol(symbol.mid(comma1, comma2 - comma1));
+            result += QLatin1Char('>');
+
+            end = findSameDepth(symbol, 0, QLatin1Char('>'), true);
+            symbol = symbol.mid(end);
+        }
+    }
+    else if (startsWith(symbol, QLatin1String("allocator<"), &end)) {
```

Comment:
can you please explain in code comments what the difference between these three conditional branches is? the comment could be as simple as

// translates `set<...>` to `set<...>`

and similar for the three cases, such that it becomes clear what the branches do conceptually without having to grog the code first